lib/game_cycle.rb

```diff
@@ -0,0 +1,8 @@
+module Game_cycle
+    def game_cycle
```

Comment:
Dont directly call the global game object in here.. Instead pass as an argument a game variable as such:
`def game_cycle(game)`
That means when you call this method in main.rb you should be passing your initialized game object:

`Game_cycle.game_cycle(game)`